tailwind.config.ts

```diff
@@ -7,6 +7,8 @@ module.exports = {
         devstation: {
           "green-100": "#114C03",
           "green-200": "#498A39",
+          "green-300": "#3B7227B2",
```

Comment:
```suggestion
          "green-300": "#3B7227B2",
```
check your hex code here, this is an invalid hex code, they should always be six characters long

Reply:
I got these values from the figma provided. Hex color codes can be 6 or 8 characters long. The last 2 characters in case of 8 characters are transparency values. See the example here: https://www.w3schools.com/cssref/tryit.php?filename=trycss_color_hexa